packages/components/src/passengerInfoItineraryCard/PassengerInfoItineraryCard.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { StyleSheet, Text } from '@kiwicom/universal-components';
+
+type PassengerInfoItineraryCardProps = {|
+  +adults: number,
+  +infants: number,
+|};
+
+export default function PassengerInfoItineraryCard({
+  adults,
+  infants,
+}: PassengerInfoItineraryCardProps) {
+  return (
+    <View style={styles.container}>
```

Comment:
<img width="387" alt="Screenshot 2019-05-29 at 13 47 04" src="https://user-images.githubusercontent.com/2660330/58555379-9bba9380-8219-11e9-87fb-5cfa6fc03d38.png">

Looks good, only on mobile the layout is slightly broken.

Reply:
To be honest I'm not sure where to actually put that, because space on mobile card is slightly limited. Maybe in line on top or under price?

Reply:
Under the price looks good to me, will move it there

Reply:
@aalises 

> Under the price looks good to me, will move it there

Sounds good to me, let's try 
THNKS

Reply:
Definitely better :)
<img width="345" alt="Screenshot 2019-05-29 at 15 22 52" src="https://user-images.githubusercontent.com/11021461/58560594-b98df580-8225-11e9-8757-56c2b741491e.png">

Reply:
Is it look OK also with infants?

Reply:
I guess so
<img width="341" alt="Screenshot 2019-05-29 at 15 34 34" src="https://user-images.githubusercontent.com/11021461/58561398-4ab19c00-8227-11e9-90b5-c8dc28e3a603.png">

Reply:
Hm not bad, just thinking if it won't be better in a row?

Reply:
Yep I was thinking about it, separated by `,`, I will do that now (only in mobile)

Reply:
I'll be also for row, to save little bit of space. Thanks

Reply:
Done @FilipMessa @JoseDuSV 
![Uploading Screenshot 2019-05-29 at 16.01.07.png…]()

Reply:
Done @FilipMessa @JoseDuSV 
<img width="344" alt="Screenshot 2019-05-29 at 16 01 07" src="https://user-images.githubusercontent.com/11021461/58563413-5b641100-822b-11e9-936a-52bc47822a29.png">